presto-tests/src/main/java/com/facebook/presto/tests/sql/PrestoSqlExecutor.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests.sql;
+
+import com.facebook.presto.testing.QueryRunner;
+
+public class PrestoSqlExecutor
+        implements SqlExecutor
+{
+    private final QueryRunner queryRunner;
```

Comment:
So that we have a common interface over jdbc drivers and presto query runner and can define `DataSetup`-s in a backend-agnostic way. So once we have `insert` support in postgres connector we can `new CreateAndInsert(prestoExecutor, 'postgres.table')`